### PR TITLE
Layout v2

### PR DESCRIPTION
**Pull request by cindy1u0**
Layout second version based on figma design

### CODE REVIEW

File: frontend/src/components/Layout.js [hunk 1]
```diff
@@ -10,38 +9,16 @@ const Wrapper = styled.div`
 const Content = styled.main`
   height: 100vh;
   margin: 0 auto;
-  padding: 5em 2em;
+  padding: 7em 5em;
```
Comment (stefan-philip) [inline frontend/src/components/Layout.js]: when I saw the preview, it seemed like the Content was not left aligned with the "vocab buddy" which is what I think was in the Figma

File: frontend/src/components/Layout.js [hunk 2]
```diff
@@ -10,38 +9,16 @@ const Wrapper = styled.div`
...
-  margin-left: 2em;
-`;
-
 // Main layout of the project that includes both header and navbar
 const Layout = (props) => {
   return (
     <Wrapper>
       <Nav />
       <Content>{props.children}</Content>
-      <Footer>
```
Comment (stefan-philip) [inline frontend/src/components/Layout.js:38]: no footer :(

Comment (stefan-philip) [reply frontend/src/components/Layout.js:38]: this is not an approval/disproval I'm just gonna miss the made by students at Sandbox

Comment (cindy1u0) [reply frontend/src/components/Layout.js:38]: hahahah i moved it to the Footer component cuz right now we don't need it anymore hehe

File: frontend/src/components/Layout.js (hunk 1 above)
Comment (cindy1u0) [reply frontend/src/components/Layout.js]: Ohh oh maybe ill change that heheh thanks for catching it

File: frontend/src/components/Layout.js [hunk 3]
```diff
@@ -10,38 +9,16 @@ const Wrapper = styled.div`
...
-
-const FooterContent = styled.div`
-  align-items: center;
-  display: flex;
-  height: 100%;
-  margin-left: 2em;
-`;
-
 // Main layout of the project that includes both header and navbar
 const Layout = (props) => {
```
Comment (jrosen081) [inline frontend/src/components/Layout.js]: Should you destructure this into ({ children })?

**Review by jrosen081 — APPROVED** [review]
Sad that the footer is leaving :(

File: frontend/src/components/Layout.js (hunk 3 above)
Comment (cindy1u0) [reply frontend/src/components/Layout.js]: OMG LMFAO i actually dk how i forgot to do it. Thanks good catch

**Review by stefan-philip — APPROVED** [review]

**Pull request merged**